src/figuresClasses.ts

```diff
@@ -1,19 +1,108 @@
 export interface Figure {
+  color: string;
+  shape: string;
 
+  getArea(): number;
 }
 
-export class Triangle {
+function isTrianglePossible(a: number, b: number, c:number): boolean {
+  const sum = [a, b, c];
+  const largest = Math.max(a, b, c);
 
+  for (let i = 0; i < 3; i += 1) {
```

Comment:
Need rewrite more understandable: use `filter` method instead `for` loop and give descriptive name for received variable